src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
```

Comment:
The `<form>` tag should not be self-closing. In HTML, form tags should have an opening and a closing tag. Please change `<form />` to `<form>`.